buyer-api/src/facades/dataResponseFacade/increaseApproval.js

```diff
@@ -18,39 +13,24 @@ const getAllowance = async myAddress =>
   wibcoin.methods.allowance(myAddress, dataExchange.options.address).call();
 
 const checkAllowance = async () => {
-  logger.info('Allowance Check :: Started');
-  const { address } = await signingService.getAccount();
-  const allowance = await getAllowance(address);
+  const account = await signingService.getAccount();
```

Comment:
Why are we getting the account object now instead of just the address? `transactionQueue` is extracting it anyway `const { address } = account;`

Reply:
Good point! I did this thinking on the `wallet parallelization` epic where transactions no longer use the address alone and we will need a processing queue by sub-account

Reply:
Great! Then that's ok 👍